tests/test_unit_tests.py

```diff
@@ -322,13 +322,13 @@ def test_remote_sign(self):
             builder = Builder(self.manifestDefinition)
             builder.set_no_embed()
```

Comment:
typically remote signing would also set_remote_url() If remote fetch is disabled the reader will return an error indicating the RemoteUrl that should be read from. Setting no_embed on its own is mainly there for writing sidecars.